dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,15 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        if (ident.spelling == "assert")
```

Comment:
We would need to be able to determine if it's reachable 'through' the `assert.h` header...